modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/wal/IgniteWalRecoveryTest.java

```diff
@@ -1529,7 +1529,8 @@ else if (rec instanceof PageDeltaRecord) {
                             int realPageSize = data.length;
 
                             pageIO.compactPage(GridUnsafe.wrapPointer(bufPtr, realPageSize), buf, realPageSize);
-                            pageIO.compactPage(ByteBuffer.wrap(data), bufWal, realPageSize);
+                            ByteBuffer dataBuf = ByteBuffer.wrap(data).order(ByteOrder.nativeOrder());
```

Comment:
Please inline

Reply:
Inlined, Not sure that it became better: now we have a line wrap and so still 2 lines. Should I leave it like this, what do you think?